spec/models/authority_spec.rb

```diff
@@ -1,6 +1,16 @@
 require 'spec_helper'
 
 describe Authority do
+  describe "validations" do
+    it 'should ensure a unique short_name' do
+      @a1 = create(:authority, short_name: "Existing")
+      @a2 = build(:authority, short_name: "Existing")
```

Comment:
This can be a local variable. Elsewhere in the file instance variables are assigned in a `before` block for use in a few examples.

It would also be good to assign it a useful name like 'new_authority' (maybe?) to help tell the story of the test.